Tropos/Controllers/TRAppDelegate.m

```diff
@@ -38,6 +40,32 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
     return YES;
 }
 
+- (void)application:(UIApplication *)application didRegisterForRemoteNotificationsWithDeviceToken:(NSData *)deviceToken;
+{
+    NSString *channel = [[NSTimeZone localTimeZone] name];
+    [self.courier subscribeToChannel:channel withToken:deviceToken];
+}
+
+- (void)application:(UIApplication *)application didReceiveRemoteNotification:(NSDictionary *)userInfo fetchCompletionHandler:(nonnull void (^)(UIBackgroundFetchResult))completionHandler;
+{
+    if (![userInfo[@"aps"][@"content-available"] isEqual: @1]) {
+        completionHandler(UIBackgroundFetchResultNoData);
+        return;
+    }
+    [self.applicationController.conditionsDescription subscribeNext:^(NSAttributedString *conditions) {
```

Comment:
I was hesitant to include UIKit here. 

Can you give an example of take? I'm not super familiar with RAC.